securedrop/source_app/main.py

```diff
@@ -257,7 +257,7 @@ def submit() -> werkzeug.Response:
                 current_app.logger.info("generating key, entropy: {}".format(
                     entropy_avail))
             else:
-                current_app.logger.warn(
+                current_app.logger.warning(
```

Comment:
Not related to this PR itself, but it seemed like `warn` was deprecated.
<img width="837" alt="Screenshot_2020-10-16_22 42 08" src="https://user-images.githubusercontent.com/4163701/96326698-da184480-1000-11eb-8226-2cdc93a93898.png">